astropy/table/row.py

```diff
@@ -164,13 +160,11 @@ def __repr__(self):
     def __unicode__(self):
         index = self.index if (self.index >= 0) else self.index + len(self._table)
         return '\n'.join(self.table[index:index + 1].pformat(max_width=-1))
-    if not six.PY2:
-        __str__ = __unicode__
+
+    __str__ = __unicode__
```

Comment:
I don't think python3 uses `__unicode__` anymore. Couldn't we just rename `__unicode__` to `__str__`?

Reply:
When I am sweeping through the dev doc (in a separate PR), there is mention of some setting that might still use this. See http://astropy.readthedocs.io/en/stable/development/codeguide.html#unicode-guidelines but don't change it here.